ui/src/jobs/components/JobItem.jsx

```diff
@@ -69,6 +72,11 @@ class JobItem extends Component {
             Posted <DateFromNow date={created} />
           </Col>
         </Row>
+        <Row className="mt3" type="flex" justify="left">
+          <Col>
+            <EditRecordActionContainer href={`${EDIT_JOB}/${recordId}`} canDisplay={ canEdit }/>
```

Comment:
This is where actions like this one go. (you need to use `rightActions` to put it on the right side)

https://github.com/inspirehep/inspirehep/blob/65486511885301b94cf9b40755410e23654b1fc5/ui/src/literature/components/LiteratureItem.jsx#L46